client/my-sites/sidebar-unified/menu.jsx

```diff
@@ -60,38 +60,40 @@ export const MySitesSidebarUnifiedMenu = ( {
 	}, [ selected, childIsSelected, reduxDispatch, sectionId, sidebarCollapsed ] );
 
 	return (
-		<ExpandableSidebarMenu
-			onClick={ () => {
-				if ( link ) {
-					if ( isExternal( link ) ) {
-						// If the URL is external, page() will fail to replace state between different domains
-						externalRedirect( link );
-						return;
+		<li>
```

Comment:
This change is just the new `<li>` right?

Reply:
Correct!

Reply:
And a fullstop 😁